backend/src/backend/main.py

```diff
@@ -97,25 +103,37 @@ async def add_book(book_payload: BookAddSchema):
                 kind=KIND_TEXT,
             )
 
-    title = await Title.objects.get_or_create(
-        ident=get_ident_from_name(book_payload.metadata["Name"])
-    )
-    await book.update(title=title)
+    for metadata_name, value in book_payload.metadata.items():
+        if metadata_name.startswith("Illustration_"):
+            await TitleMetadata.objects.create(
+                title=title.ident,
+                name=metadata_name,
+                bin_value=base64.standard_b64decode(value),
+                kind=KIND_ILLUSTRATION,
+            )
+        else:
+            await TitleMetadata.objects.create(
+                title=title.ident,
+                name=metadata_name,
+                value=value,
+                kind=KIND_TEXT,
+            )
 
     for tag_name in book_payload.metadata["Tags"].split(";"):
         book_tag = await BookTag.objects.get_or_create(name=tag_name)
         await book.tags.add(book_tag)
-        await title.tags.add(book_tag)
+        if not tag_name.startswith("_") or "_category" in tag_name:
```

Comment:
Oh come on! I **specifically** requested you exclude the list I gave you. It's not even correct as `_category` tag doesn't exist. It's a prefix that we use like `_category:ted` or `_category:wikipedia`.